### PR TITLE
OFFAPPS-535 fixed newlines for agents (not admins)

### PR DESCRIPTION
**Pull request by ocke**
fixed newlines for agents (not admins)

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-535

### Risks
 - None

### CODE REVIEW

File: templates/display.hdbs [hunk 1]
```diff
@@ -73,7 +73,7 @@
           {{#if editable}}
             <textarea class="notes-or-details" data-field-name="{{name}}" data-field-type="users">{{value}}</textarea>
           {{else}}
-            <p>{{#if html}}{{{value}}}{{else}}{{value}}{{/if}}</p>
+            <p>{{{value}}}</p>
```
Comment (gfxmonk) [inline templates/display.hdbs]: This is going to interpret _all_ values as html (even the branches that weren't previously). Suggest reinstating the `.html` flag, and setting it to true in your new `if` branch above.

File: app.js [hunk 2]
```diff
@@ -181,9 +182,8 @@
           if (field.type === 'date') {
             result.value = (result.value ? this.toLocaleDate(result.value) : '');
           }
-          else if(field.type === 'textarea' && values[key] != null) {
+          else if(!result.editable && !!values[key]) {
             result.value = _.escape(values[key]).replace(/\n/g,'<br>');
```
Comment (gfxmonk) [inline app.js]: It'd be nice if this repeated code could be pulled out, but it's not a biggie.

**Comment by tobico** [comment]
:+1: after Tim's comments addressed

**Comment by ocke** [comment]
@gfxmonk can you have a look again?

**Comment by gfxmonk** [comment]
:+1: (but squash it before merging)

**Pull request merged**